ioctl/cmd/bc/bc.go

```diff
@@ -53,6 +53,7 @@ func init() {
 	BCCmd.AddCommand(bcBlockCmd)
 	BCCmd.AddCommand(bcInfoCmd)
 	BCCmd.AddCommand(bcBucketListCmd)
+	BCCmd.AddCommand(bcBLCmd)
```

Comment:
think we could modify the existing bcBucketListCmd, that is:

ioctl bc bucketlist voter io1xxx --> this is current bcBucketListCmd
ioctl bc bucketlist cand io1xxx --> this is new bucket by candidate command

"ioctl bc bucketlist io1xxx" would fail, but that's OK, user can easily check the right way to use by typing in "ioctl bc bucketlist" and see the help